src/App.vue

```diff
@@ -95,38 +111,7 @@ export default {
 </script>
 
 <style>
-body, html {
-  width: 100%;
-  height: 100%;
-  margin: 0;
-  padding: 0;
-}
-
-::-webkit-scrollbar {
-  width: 0px;  /* Remove scrollbar space */
-  background: transparent;  /* Optional: just make scrollbar invisible */
-}
-</style>
-
-<style scoped>
-  #app {
-    display: grid;
-    grid-template-columns:
-    22rem 1fr;
-    grid-template-areas:
-    "sidebar body";
-  }
-  .sidebar{
-    grid-area: sidebar;
-    background-color:rgb(34, 44, 50);
-  }
-  .body { grid-area: body; } 
-  .sidebar { grid-area: sidebar; }
-  .results-enter-active, .results-leave-active {
-    transition: all 0.2s ease-out;
-  }
-  .results-enter, .results-leave-active {
-    opacity: 0;
-    transform: translateY(-30px);
+  html, body {
```

Comment:
did we want to apply this globally?

Reply:
We might not need that styling because of Vuetify's magic come to think of it, we could remove it and see what happens 😸